docs/tuf-spec.txt

```diff
@@ -754,30 +780,28 @@ Version 1.0 (Draft)
    in a delegation chain, not just in a trusted path of the role that describes
-   the target file.  The format of a PATHPATTERN may be either a path to a
-   single file, or a path to a directory to indicate all files and/or
-   subdirectories under that directory.
-
-   A path to a directory is used to indicate all possible targets sharing that
-   directory as a prefix; e.g. if the directory is "targets/A", then targets
-   which match that directory include "targets/A/B.json" and
-   "targets/A/B/C.json".
-
-   We are currently investigating a few "priority tag" schemes to resolve
-   conflicts between delegated roles that share responsibility for overlapping
-   target paths. One of the simplest of such schemes is for the client to
-   consider metadata in order of appearance of delegations; we treat the order
-   of delegations such that the first delegation is trusted more than the
-   second one, the second delegation is trusted more than the third one, and so
-   on. The metadata of the first delegation will override that of the second
-   delegation, the metadata of the second delegation will override that of the
-   third delegation, and so on. In order to accommodate this scheme, the
-   "roles" key in the DELEGATIONS object above points to an array, instead of a
-   hash table, of delegated roles.
-
-   Another priority tag scheme would have the clients prefer the delegated role
-   with the latest metadata for a conflicting target path. Similar ideas were
-   explored in the Stork package manager (University of Arizona Tech Report
+   the target file.  PATHPATTERN can include shell-style wildcards and supports
+   the Unix filename pattern matching convention.  Its format may either
+   indicate a path to a single file, or to multiple paths with the use of
+   shell-style wildcards.  For example, the path pattern "targets/*.tgz" would
+   match file paths "targets/foo.tgz" and "targets/bar.tgz", but not
+   "targets/foo.txt".  Likewise, path pattern "foo-version-?.tgz" matches
+   foo-version-2.tgz", but not "foo-version-alpha.tgz".
+
+   Several schemes exist to resolve conflicts between delegated roles that
+   share responsibility for overlapping target paths. One of the simplest of
+   such schemes is for the client to consider metadata in order of appearance
+   of delegations; we treat the order of delegations such that the first
+   delegation is trusted more than the second one, the second delegation is
+   trusted more than the third one, and so on. The metadata of the first
+   delegation will override that of the second delegation, the metadata of the
+   second delegation will override that of the third delegation, and so on. In
+   order to accommodate this scheme, the "roles" key in the DELEGATIONS object
+   above points to an array, instead of a hash table, of delegated roles.
+
+   Another scheme would have the clients prefer the delegated role with the
```

Comment:
I think we can remove this paragraph. It's an interesting, historical footnote, but has nothing to do with how TUF works now.